### ISSUE TITLE
GStreamer cache upgrade problems

### ISSUE THREAD

**Issue by lazka** [quodlibet/quodlibet#2839]
We are currently blocked on https://bugzilla.gnome.org/show_bug.cgi?id=795750 (for Windows at least)

Alternatively we can try to force a cache regen when the user upgrades.

**Comment by lazka** [comment]
Another plan: In case there is a registry file at specific path in the tree, set GST_REGISTRY and GST_REGISTRY_UPDATE="no". And generate the file in the windows installer script.
